conf/application.conf

```
@@ -23,8 +23,8 @@ play {
     """
   }
   filters.headers {
-    # Unset some of the security filters enabled in datastore.Filters
-    contentSecurityPolicy = "default-src 'self'; script-src 'self' 'unsafe-eval' www.googletagmanager.com 'sha256-NA81o2FV8jh0UizSPsBz2qEGpXfap54Eghuo7vxrH8g='; style-src 'self' 'sha256-DOu86drLfwUr1Wcsx/wxfqAogK7tFvJGjVmF/300H/M=' 'sha256-47DEQpj8HBSa+/TImW+5JCeuQeRkm5NMpJWZG3hSuFU='; connect-src 'self' ws://localhost:9002 app.ovly.co; frame-src webknossos.olvy.co; img-src 'self' data:"
+    # The sha256 hash in the script-src refers to the google analytics configuration script in main.scala.html. 
+    contentSecurityPolicy = "default-src 'self'; script-src 'self' 'unsafe-inline' 'unsafe-eval' *.googletagmanager.com  app.olvy.co webknossos.olvy.co; style-src 'self' 'unsafe-inline'; connect-src 'self' ws://localhost:9002 app.olvy.co api.github.com gist.github.com *.google-analytics.com api.airbrake.io; frame-src webknossos.olvy.co; img-src 'self' data:"
```

Note to myself: As far as I understood, unsafe-inline shouldn't be allowed as a script source, but only as a style-src, because of olvy.

```suggestion
    contentSecurityPolicy = "default-src 'self'; script-src 'self' 'unsafe-eval' *.googletagmanager.com  app.olvy.co webknossos.olvy.co; style-src 'self' 'unsafe-inline'; connect-src 'self' ws://localhost:9002 app.olvy.co api.github.com gist.github.com *.google-analytics.com api.airbrake.io; frame-src webknossos.olvy.co; img-src 'self' data:"
```

---

According to [this comment](https://github.com/scalableminds/webknossos/pull/7367#issuecomment-1757717225) unsafe-inline is needed as a script-src in firefox to use the react dev tools. Shouldn't be used in production, though.